docs(theme): Get Started in Style

paulftw opened the pull request:
Fixes GQLSAAS-741

<img width="1492" alt="Screen Shot 2020-10-13 at 12 28 11 PM" src="https://user-images.githubusercontent.com/769142/95842939-d27e4480-0d4f-11eb-857a-ee9038e193d6.png">

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/dgraph-io/dgraph/6713)
<!-- Reviewable:end -->
 
 
<!-- Dgraph:start -->
Docs Preview: [<img src="https://bl.ocks.org/prashant-shahi/raw/3a9f99bec84231cfe3c0e82cf883f159/0e588d908ad8c8b10958b87ebdd2ba68779ccf4f/dgraph.svg" height="34" align="absmiddle" alt="Dgraph Preview"/>](https://dgraph-3511eefc04-101326.surge.sh)
<!-- Dgraph:end -->

bucanero commented on wiki/content/_index.md:
```diff
@@ -9,162 +9,134 @@ aliases = ["/contribute"]
+    </a>
+  </div>
+  <div class="item">
+    <div class="icon"></div>
```
Hi @paulftw ,
could you add an icon to the Tutorials card? thanks!

paulftw replied:
done

bucanero reviewed (changes requested):
The new page looks great.

I left a comment to add a missing icon.

Also, the title styling for `Clients`, `Deploy`, `Enterprise features`, `FAQ`, and `Badger` cards look different than the rest.  Other titles are black and bold, but these are regular and blue. I assume it's just a CSS style fix.

paulftw commented:
@bucanero addressed your comments.

In this PR I only focused on getting the designer's style codified. 
Microcopy, icons, section order, etc is something that DevRel or Docs teams can modify any time they like.

bucanero commented:
> @bucanero addressed your comments.
> 
> In this PR I only focused on getting the designer's style codified.
> Microcopy, icons, section order, etc is something that DevRel or Docs teams can modify any time they like.

thanks @paulftw , yes you're right, we can take it from here. Thanks again for your help!
